README.md

```diff
@@ -191,6 +191,27 @@ You can specify a swagger nickname to use instead of the auto generated name by
 desc 'Get a full list of pets', nickname: 'getAllPets'
 ```
 
+## Expose nested namespace as standalone route
+Use the `nested = false` option to expose nested actions as standalone resources.
```

Comment:
Probably should say `nested: false` rather than `nested = false` since we're Ruby.